cmd/restic/cmd_dump.go

```diff
@@ -58,30 +61,7 @@ func splitPath(p string) []string {
 }
 
 func dumpNode(ctx context.Context, repo restic.Repository, node *restic.Node) error {
```

Comment:
Remove `dumpNode()` and call `getNodeData()` at line 83 (the only place where it is called).